src/Java.Interop/Java.Interop.csproj

```diff
@@ -4,8 +4,19 @@
     <XAConfigPath>..\..\bin\Build$(Configuration)\XAConfig.props</XAConfigPath>
   </PropertyGroup>
   <Import Condition="Exists ('$(XAConfigPath)')" Project="$(XAConfigPath)" />
+  <!--
+    NOTE: in xamarin-android, this project gets built by xabuild in Xamarin.Android-Tests.sln
+    Exclude net6.0, because xabuild cannot build net5.0 or higher
+  -->
+  <PropertyGroup Condition=" '$(XABuild)' == 'true' ">
```

Comment:
What sets `$(XABuild)`?  I don't see any "obvious" references within xamarin-android:

```
% git grep '\<XABuild\>'
tools/xabuild/XABuild.cs:       class XABuild
```

Is this via some MSBuild magic?

Reply:
It is magic, this XML file we generate presets MSBuild properties:

https://github.com/xamarin/xamarin-android/pull/5895/files#diff-3b8f9e4db31a6f4b54073ea66ded003fa77ec1573d42406911e0e3b7a691acbcR122

So I made any `xabuild` invocation set this property by default.

Reply:
it's from: https://github.com/xamarin/xamarin-android/pull/5895/files#diff-3b8f9e4db31a6f4b54073ea66ded003fa77ec1573d42406911e0e3b7a691acbcR122